### PR TITLE
chore: votes inspector for all validators  

### PR DESCRIPTION
**Pull request by gsk967**
If there is no validators list on config, inspector will check the votes for all validators 

Example config:
```yaml
---
validators:
window: 20
networks:
  canon:
    rpc: https://canon-4.rpc.network.umee.cc:443
    grpc: tcp://canon-4.network.umee.cc:11000
  mainnet:
    rpc: http://umee.rpc.m.stavr.tech:10457
    grpc: umee-grpc.polkachu.com:13690
  local:
    rpc: http://localhost:26657
    grpc: http://localhost:9090

```

### CODE REVIEW

File: inspector_go/inspector.go [hunk 1]
```diff
@@ -34,9 +34,18 @@ func StartInspector(cfg config.Configuration, rpcUri, grpcUri string) error {
...
+		for _, val := range validators {
+			votes[val.OperatorAddress] = config.NewValidatorVotes(val.OperatorAddress, expectedNoOfVotes)
+			vInfo[val.OperatorAddress] = val.GetMoniker()
+		}
+	} else {
+		for _, v := range cfg.Validators {
+			votes[v.Validator] = config.NewValidatorVotes(v.Validator, expectedNoOfVotes)
+			vInfo[v.Validator] = v.Moniker
+		}
 	}
```
Comment (robert-zaremba) [inline inspector_go/inspector.go:49]:
```suggestion
	validators := cfg.Validators
	if len(cfg.Validators) == 0 {
		log.Println("ℹ️ Getting all staking validators...")
		validators = rpc.GetValidators(codec, grpcUri)
	}
		for _, val := range validators {
			votes[val.OperatorAddress] = config.NewValidatorVotes(val.OperatorAddress, expectedNoOfVotes)
			vInfo[val.OperatorAddress] = val.GetMoniker()
		}
```

File: inspector_go/main.go [hunk 2]
```diff
@@ -29,17 +30,25 @@ func main() {
...
 				rpc = cfg.Networks.Mainnet.RPC
 				grpc = cfg.Networks.Mainnet.GRPC
+			case "canon":
+				rpc = cfg.Networks.Canon.RPC
+				grpc = cfg.Networks.Canon.GRPC
+			case "local":
+				rpc = cfg.Networks.Local.RPC
+				grpc = cfg.Networks.Local.GRPC
+			default:
+				log.Fatalf("this netwokr %s is not supported", network)
```
Comment (robert-zaremba) [inline inspector_go/main.go:45]:
```suggestion
				log.Fatalf("this network %s is not supported", network)
```

File: inspector_go/main.go [hunk 3]
```diff
@@ -29,17 +30,25 @@ func main() {
...
 			}
 			return StartInspector(cfg, rpc, grpc)
 		},
 	}
 
-	rootCmd.Flags().String(config.Network, "canon", "Network to use (canon/mainnet)")
+	rootCmd.Flags().String(config.Network, "canon", "Network to use (canon/mainnet/local)")
```
Comment (robert-zaremba) [inline inspector_go/main.go:51]:
```suggestion
	rootCmd.Flags().String(config.Network, "canon", "Network to use (canon, mainnet or local)")
```

**Review by robert-zaremba — APPROVED** [review]

File: inspector_go/inspector.go (hunk 1 above)
Comment (gsk967) [reply inspector_go/inspector.go:49]: `cfg.Validators` and `validators` are not same type

**Pull request merged**